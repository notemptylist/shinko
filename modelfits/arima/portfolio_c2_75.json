{
    "numlags": 400,
    "results": [
        {
            "mean": -0.0023880867208311957,
            "order": [
                0,
                2,
                2
            ],
            "rmse": 4.169788478242659,
            "tstamp": 1639810350.0502465
        },
        {
            "mean": -0.0023880867208311957,
            "order": [
                0,
                0,
                2
            ],
            "rmse": 4.203003917785424,
            "tstamp": 1639810350.0502465
        },
        {
            "mean": -0.0023880867208311957,
            "order": [
                1,
                0,
                0
            ],
            "rmse": 4.3232870007547985,
            "tstamp": 1639810350.0502465
        }
    ],
    "stream": "portfolio_c2_75.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            2
        ],
        [
            0,
            2,
            0
        ],
        [
            0,
            2,
            1
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            2
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            2
        ],
        [
            1,
            2,
            0
        ],
        [
            1,
            2,
            1
        ],
        [
            1,
            2,
            2
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            2
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            2
        ],
        [
            2,
            2,
            0
        ],
        [
            2,
            2,
            1
        ],
        [
            2,
            2,
            2
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            2
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            2
        ],
        [
            3,
            2,
            0
        ],
        [
            3,
            2,
            1
        ],
        [
            3,
            2,
            2
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            2
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            2
        ],
        [
            6,
            2,
            0
        ],
        [
            6,
            2,
            1
        ],
        [
            6,
            2,
            2
        ]
    ],
    "version": "0.1"
}