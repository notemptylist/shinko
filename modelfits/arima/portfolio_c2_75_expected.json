{
    "numlags": 400,
    "results": [
        {
            "mean": -0.006472271437363532,
            "order": [
                5,
                0,
                0
            ],
            "rmse": 0.534364507083923,
            "tstamp": 1640244764.1516056
        },
        {
            "mean": -0.006472271437363532,
            "order": [
                4,
                1,
                1
            ],
            "rmse": 0.549256941629726,
            "tstamp": 1640244764.1516056
        },
        {
            "mean": -0.006472271437363532,
            "order": [
                0,
                3,
                0
            ],
            "rmse": 1.6302622865172902,
            "tstamp": 1640244764.1516056
        },
        {
            "mean": -0.02495682908715745,
            "order": [
                0,
                1,
                1
            ],
            "rmse": 0.5364376463584345,
            "tstamp": 1640257368.4419634
        },
        {
            "mean": -0.02495682908715745,
            "order": [
                2,
                0,
                3
            ],
            "rmse": 0.6192712343343824,
            "tstamp": 1640257368.4419634
        },
        {
            "mean": -0.02495682908715745,
            "order": [
                4,
                1,
                0
            ],
            "rmse": 0.6315070897851477,
            "tstamp": 1640257368.4419634
        },
        {
            "mean": -0.07332309490114323,
            "order": [
                3,
                0,
                0
            ],
            "rmse": 0.31473414845935427,
            "tstamp": 1640367401.0810924
        },
        {
            "mean": -0.07332309490114323,
            "order": [
                1,
                1,
                3
            ],
            "rmse": 0.32070874748104017,
            "tstamp": 1640367401.0810924
        },
        {
            "mean": -0.07332309490114323,
            "order": [
                4,
                3,
                0
            ],
            "rmse": 0.567477647246155,
            "tstamp": 1640367401.0810924
        }
    ],
    "stream": "portfolio_c2_75_expected.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}